WebHost.py

```diff
@@ -37,11 +37,16 @@ def get_app() -> "Flask":
     parser = argparse.ArgumentParser(allow_abbrev=False)
     parser.add_argument('--config_override', default=None,
                         help="Path to yaml config file that overrules config.yaml.")
+    parser.add_argument('--port'default=None,
```

Comment:
port is already part of the  config file

Reply:
Yes but there didn't seem to be a way to pass it in via the command line a la `python WebHost.py --port 1234` as required by the HttpPlatformHandler.